nuxt.config.ts

```diff
@@ -26,19 +26,19 @@ export default defineNuxtConfig({
   },
   runtimeConfig: {
     public: {
-      SENTRY_DSN: process.env.SENTRY_DSN,
-      APP_DOMAIN: process.env.APP_DOMAIN,
-      MIDDLEWARE_URL: process.env.MIDDLEWARE_URL,
-      NODE_URL: process.env.NODE_URL,
-      WEBSOCKET_URL: process.env.WEBSOCKET_URL,
-      DEX_BACKEND_URL: process.env.DEX_BACKEND_URL,
-      NETWORK_NAME: process.env.NETWORK_NAME,
-      ALTERNATIVE_NETWORK_URL: process.env.ALTERNATIVE_NETWORK_URL,
-      ALTERNATIVE_NETWORK_NAME: process.env.ALTERNATIVE_NETWORK_NAME,
-      AE_TOKEN_ID: process.env.AE_TOKEN_ID,
-      DEBUG_MODE: process.env.DEBUG_MODE,
-      CONTRACT_VERIFICATION_SERVICE_URL: process.env.CONTRACT_VERIFICATION_SERVICE_URL,
-      SH_DEX_CONTRACTS: process.env.SH_DEX_CONTRACTS?.split(';'),
+      SENTRY_DSN: undefined,
+      APP_DOMAIN: undefined,
+      MIDDLEWARE_URL: undefined,
+      NODE_URL: undefined,
+      WEBSOCKET_URL: undefined,
+      DEX_BACKEND_URL: undefined,
+      NETWORK_NAME: undefined,
+      ALTERNATIVE_NETWORK_URL: undefined,
+      ALTERNATIVE_NETWORK_NAME: undefined,
+      AE_TOKEN_ID: undefined,
+      DEBUG_MODE: undefined,
+      CONTRACT_VERIFICATION_SERVICE_URL: undefined,
+      SH_DEX_CONTRACTS: undefined,
```

Comment:
As I understand `runtimeConfig.public` is made to define config entries and optionally provide default values. These values can be overriden by `NUXT_PUBLIC_<NAME>`.
```
SH_DEX_CONTRACTS: process.env.SH_DEX_CONTRACTS?.split(';'),
```
meaning mixing `SH_DEX_CONTRACTS` vs `NUXT_PUBLIC_SH_DEX_CONTRACTS` and in the first case it would be an array, and a string in the second.

Reply:
The intention behind this, that some variables will be picked up from stage-specific env config
`NUXT_PUBLIC_` prefix is Nuxt aut magic

Reply:
Do you have a case when you need provide variables without `NUXT_PUBLIC_`? As I see these prefixed variables would work in any case, no need to access for example `process.env.SENTRY_DSN` directly.